src/groundStation/uTransceiver.py

```diff
@@ -0,0 +1,133 @@
+        print('Received from UHF:') 
+        self.listen_en = True
+        start = time.time()
+        while (time.time() - start) < self.listentimeout_s:
+            #print('here')
+            # data = s.recv(10000)
+            # print(repr(data))
+            
+
+            if dict(poller.poll())[socket] == zmq.POLLIN:
+                print('here')
+                #print(socket.recv(zmq.DONTWAIT))
+            
+
+        socket.disconnect("tcp://localhost:%s" % port)
+        # s.close()
+
+    def enterPipeMode(self):
+            #current config is for RF mode 5, baudrate = 115200
+            self.UHFDIRCommand('UHFDIR_genericWrite(0, 0 3 0 5 0 0 1 0 0 0 1 1)')
+
+
+    def UHFDIRCommand(self, string):
+        if self.u == True:
+            cmd = (string.split('_')[1]).split('(')[0]
+            cmdcode = string.split(',')[0]
+            cmdcode = int(cmdcode.split('(')[1])
+
+            param = string.split(',')[1]
+            param = param.split(')')[0]
+
+            #parse param into correct ctypes based on cmdcode
+            if cmdcode == 0:
+                paramlist = param.split()
+                paramlist = list(map(int, paramlist))
+                arg = (ctypes.c_ubyte * len(paramlist))(*paramlist)#is this a pointer or just a list?
+                
+                #arg = ctypes.cast(args, ctypes.POINTER(ctypes.c_ubyte))
+            if cmdcode == 6:
+                param = ctypes.c_uint16(param) #TODO check that this works with space after comma in command
+                arg = ctypes.cast(param, ctypes.POINTER(ctypes.c_uint16))
+            if cmdcode == 253:
+                pass
+                #TODO someday (FRAM usage)
+                            
+            #check command and call relevant functions with args
+            if (time.time() - self.last_tx_time) > self.pipetimeout_s:
+                retval = 0
+                if cmd == 'genericWrite':
+                    retval = self.uhf.UHF_genericWrite(ctypes.c_ubyte(cmdcode), arg)
+                if cmd == 'genericRead':
+                    voidptr = (ctypes.c_void_p * 1)()
+                    self.uhf.UHF_genericRead(cmdcode, voidptr)
+                if cmd == 'genericI2C':
+                    pass
+                    #TODO someday
+                    #retval = uhf.UHF_genericI2C(cmd, )
+
+                #This should be a catch-all for any syntax errors as-is
+                #TODO: make more robust handler for incorrect inputs to prevent erronious commands being sent?
+                if retval != 0:
+                    print('UHF Equipment Handler error ' + str(retval))
+
+                #self.listen()
+            else:
+                print('Error: Command not sent. Wait for pipe mode to expire')
+                print('Pipe mode timer set to ' + str(self.pipetimeout_s) + 's')
+        else:
+            print('UHF functionality not enabled. Please run CLI with -u flag to enable.')
+
+
+#typedef enum{
+#	U_GOOD_CONFIG =  0,
```

Comment:
Commented code should be removed. These values should be added to the python bindings though so you can access them in python